content/blog/2021/10/new-architecture-to-ease-kubewarden-administrators-lives.md

```diff
@@ -0,0 +1,257 @@
+---
+title: A new architecture to ease Kubewarden administrators' lives
+authors:
+    - Víctor Cuadrado Juan
+date: 2021-10-01
+---
+
+We are pleased to announce a new architecture for the Kubewarden stack, in line
+with its journey to maturity:
+
+The introduction of a **PolicyServer** Custom Resource Definition which allows
+users to describe a policy-server Deployment, together with binding
+**ClusterAdmissionPolicies** to a specific **PolicyServer** instance.
+
+These 2 changes are accompanied by a multitude of improvements to make Kubewarden
+more comfortable for Kubernetes Administrators, such as validation for
+Kuberwarden Custom Resources, improvements in Helm Charts, and Status and
+Conditions for **ClusterAdmissionPolicies**.
+
+For a comprehensive list, see the [release notes](TODO) of our latest Helm
+charts.
+
+## How does it look like?
+
+In previous versions, the Kubewarden Controller instantiated a single Deployment
+of policy-server. That policy-server was configured via a ConfigMap, which
+contained the Deployment options (image, replicas, …), and a list of policies to
+load compiled from the instantiated ClusterAdmissionPolicies, with information
+on where to pull them from, their configuration options and so on.
+
+With the addition of the new **PolicyServer** Custom Resource, administrators
+have a better UX (just define the Custom Resources), can define more than 1
+policy-server Deployment, and get to select what ClusterAdmissionPolicies target
+which PolicyServers. Let's see a diagram of the new architecture:
```

Comment:
```suggestion
With the addition of the new **PolicyServer** Custom Resource, administrators have a better UX, since they can define as many policy servers as they need, and get to select what PolicyServer each ClusterAdmissionPolicy targets. Let's see a diagram of the new architecture:
```

Just an idea of a rephrase.

Reply:
Indeed, better :).
Fixed in https://github.com/kubewarden/kubewarden.io/pull/33/commits/c2429ef94810f7aaa7cee8ed97120415ef090969.